### ISSUE TITLE
Bug: Getting an error while running analyze command for eks cluster

### ISSUE THREAD

**Issue by mannachi-az** [clowdhaus/eksup#45]
### Expected Behaviour

eksup supposed to display the analysis results for the eks cluster

### Current Behaviour

Its showing below error although i have setup kubecontext correclty.

Error: ApiError: the server could not find the requested resource: NotFound (ErrorResponse { status: "Failure", message: "the server could not find the requested resource", reason: "NotFound", code: 404 })

Caused by:
    the server could not find the requested resource: NotFound

### Code snippet

```sh
Command being used:
eksup analyze -c eks-cluster-test -r us-east-1
```

### Steps to Reproduce

Run command below for any eks cluster
eksup analyze -c eks-cluster-test -r us-east-1

### eksup version

latest

### Operating system

macOS x86_64

### Error output

```Text
Error: ApiError: the server could not find the requested resource: NotFound (ErrorResponse { status: "Failure", message: "the server could not find the requested resource", reason: "NotFound", code: 404 })

Caused by:
    the server could not find the requested resource: NotFound
```

**Comment by bryantbiggs** [comment]
You can get more details by passing `-vvv` or varying number of `v`'s for more or less logging output. Also, `eksup` currently does not support named contexts - it relies on the default context. So if you run `aws eks update-kubeconfig ...` and then run `eksup`, it should work

**Comment by mannachi-az** [comment]
[error.txt](https://github.com/clowdhaus/eksup/files/12095511/error.txt)
Attached is the error output..

**Comment by bryantbiggs** [comment]
this should be resolved now in [`v0.3.2`](https://github.com/clowdhaus/eksup/releases/tag/v0.3.2)
